wordpress-mobile/WordPress-Android
# Stats accessibility improvements 2

This PR looks rather big but it needed to be this way because I had to add the content description to all the `ListItems` and I didn't want to create a temp solution where the content description would be nullable. If you'd rather have that, I can try to split this PR into 2.

This covers all the list items in the Stats. For example the following screenshot:

<img width="485" alt="Screenshot 2019-07-24 at 15 39 02" src="https://user-images.githubusercontent.com/1079756/61798207-2f2ed000-ae29-11e9-813b-d1cfe926928b.png">

would now have the header (`Title ... Views`) marked as not important for accessibility and selecting any item in the list would read as: `Title: Home page / Archives, Views: 24`. This should help the users with keeping the context. 

There was quite a few of these list items in the Stats but all the changes should be almost the same~

To test:
* Go to Stats with Talkback turned on
* Click on all the list items to check the content description
* Check that the headers are not selectable

Update release notes:
